### PR TITLE
Some docs for IntoSystemSet

### PR DESCRIPTION
**Pull request by stepancheg**

### CODE REVIEW

File: crates/bevy_ecs/src/schedule/config.rs [hunk 1]
```diff
@@ -206,6 +206,38 @@ impl<T> NodeConfigs<T> {
 }
 
 /// Types that can convert into a [`SystemConfigs`].
+///
+/// Trait is implemented for "systems" (functions), or tuples thereof.
```
Comment (killercup) [inline crates/bevy_ecs/src/schedule/config.rs:210]:
```suggestion
/// This trait is implemented for "systems" (functions), or tuples thereof.
```

File: crates/bevy_ecs/src/schedule/config.rs [hunk 2]
```diff
@@ -221,11 +253,17 @@ where
     }
 
     /// Run before all systems in `set`.
+    ///
+    /// Note the given set is not implicitly added to the schedule when this system set is added.
```
Comment (killercup) [inline crates/bevy_ecs/src/schedule/config.rs:257]:
```suggestion
    /// Note: The given set is not implicitly added to the schedule when this system set is added.
```

File: crates/bevy_ecs/src/schedule/config.rs [hunk 3]
```diff
@@ -221,11 +253,17 @@ where
...
+    /// It is safe, but no dependencies will be created.
     fn before<M>(self, set: impl IntoSystemSet<M>) -> SystemConfigs {
         self.into_configs().before(set)
     }
 
     /// Run after all systems in `set`.
+    ///
+    /// Note the given set is not implicitly added to the schedule when this system set is added.
```
Comment (killercup) [inline crates/bevy_ecs/src/schedule/config.rs:265]:
```suggestion
    /// Note: The given set is not implicitly added to the schedule when this system set is added.
```

**Review by killercup — APPROVED** [review]

File: crates/bevy_ecs/src/schedule/config.rs [hunk 4]
```diff
@@ -206,6 +206,38 @@ impl<T> NodeConfigs<T> {
 }
 
 /// Types that can convert into a [`SystemConfigs`].
+///
+/// This trait is implemented for "systems" (functions), or tuples thereof.
```
Comment (alice-i-cecile) [inline crates/bevy_ecs/src/schedule/config.rs:210]:
```suggestion
/// This trait is implemented for "systems" (functions whose arguments all implement [`SystemParam`]), or tuples thereof.
```

**Review by alice-i-cecile — APPROVED** [review]
Definitely an improvement. Suggestion is non-blocking but I think helpful.

**Review by killercup — APPROVED** [review]

**Pull request merged**